src/app/shared/utils/utils.ts

```diff
@@ -193,11 +197,16 @@ export class Util {
     return finalMessage;
   }
 
-  public static getPersonalCabinetTitle(userRole, subrole): PersonalCabinetTitle {   
+  public static getPersonalCabinetTitle(userRole, subrole): PersonalCabinetTitle {
     if(userRole !== Role.provider){
```

Comment:
it can be also simplified to 
return (userRole !== Role.provider) ? PersonalCabinetTitle[userRole] : PersonalCabinetTitle[subrole];